README.md

```diff
@@ -28,6 +28,26 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 
 ### A
 
+#### **Accord Project**
```

Comment:
You also need to add them in the "Least Involvement to Most Involvement" section below. Please follow the Contributing.md guidelines.